### ISSUE TITLE
Backport `as` from tko.binding.foreach

### ISSUE THREAD

**Issue by brianmhunt** [brianmhunt/knockout-fast-foreach#47]
This: https://github.com/knockout/tko.binding.foreach/commit/b3380c97c7c0ff453b700d1d39e9c03ffc3d62a4

@cervengoc – If you get a chance, please feel free to incorporate the changes from the referenced patch.

Cheers.

**Comment by jensb** [comment]
Yes, can we please do this? We are using the 'as' parameter heavily in knockout-v3.5 (see also issue 52) and would love to benefit from the speedup this library provides.

**Comment by brianmhunt** [comment]
Thanks @jensb — do you think you'd be able to put together a PR?

**Comment by jensb** [comment]
I tried. Let's see if it works, shall we? :-)

**Comment by jensb** [comment]
Can you please tes this PR? I am getting errors about $parent being undefined when testing it on my app. Is this variable now obsolete (and what do I replace it with)?

**Comment by brianmhunt** [comment]
Hi @jensb I'm not set up to test this right now, but if `$parent` is missing, I'd think the problem is with one of the return values of `createContextGenerator` — in particular look to see if what they return contains `$parent`.
